hieroplatform/src/test/java/org/hiero/sketch/IntConverter.java

```diff
@@ -0,0 +1,13 @@
+package org.hiero.sketch;
```

Comment:
I don't think it's a bug, it is a reasonable behavoir  to demand that if the data type of the column is something other than double, then there must be an instantiation of the converter.